include/hal/library/cryptlib/cryptlib_ec.h

```diff
@@ -45,6 +45,21 @@ extern void libspdm_ec_free(void *ec_context);
 extern bool libspdm_ec_set_pub_key(void *ec_context, const uint8_t *public_key,
                                    size_t public_key_size);
 
+/**
+ * Sets the private key component into the established EC context.
+ *
+ *
```

Comment:
please describe the size requirement. similar the one described in `libspdm_ec_set_pub_key`.

Reply:
Thanks. I have added the private key size description.